### PR TITLE
release v1.5.1

### PR DESCRIPTION
**Pull request by wwwtyro**
* removes elastic charms
* upgrades to v1.5.1
* CNI
* etcd using easyrsa charm for TLS

### CODE REVIEW

File: bundle.yaml [hunk 1]
```diff
@@ -95,17 +56,15 @@ relations:
...
-    - "flannel:host"
   - - "kubernetes-worker:kube-api-endpoint"
     - "kubeapi-load-balancer:website"
-  - - "flannel:etcd"
-    - "etcd:db"
   - - "kubeapi-load-balancer:certificates"
     - "easyrsa:client"
-  - - "kibana:rest"
-    - "elasticsearch:client"
+  - - "flannel:etcd"
```
Comment (mbruzek) [inline bundle.yaml]: I would have expected to see an etcd -> easyrsa relation here.

Comment (chuckbutler) [inline bundle.yaml]: +1

**Comment by chuckbutler** [comment]
@mbruzek @wwwtyro have we run the bundle tests against this proposal?

**Comment by mbruzek** [comment]
So I downloaded this bundle and deployed it on aws/us-west-2. Manual tests looked great, cluster-info was fine and I was able to see pods, nodes, deployments, etc using kubectl. I downloaded the config and kubectl and was able to run commands from my laptop and even brought up the ui/dashboard as expected. The microbot action worked when I followed the instructions on the readme.

# e2e tests
After my manual tests I deployed the e2e charm with a 1.5.1 resource and made the appropriate relations. I ran the "test" action on the e2e charm and here are the results:

Ran 113 of 473 Specs in 2477.188 seconds
FAIL! -- 100 Passed | 13 Failed | 0 Pending | 360 Skipped 

@Cynerva was able to get all 113 tests to pass on his AWS deploy of the kubernetes-core. We looked at the differences and there was none on the e2e charm or the resource. Will require more investigation.

# bundletester
After the e2e tests were done I ran the bundletester. Here are the results:
 FAIL: bundle::charm-proof
[/usr/local/bin/charm-proof     exit 100]
W: No readme file found

PASS: 11 ERROR: 2 FAIL: 5 FAIL: 1 Total: 19 (1067.627708 sec)

The missing readme bundle proof error is my fault because I only downloaded the `bundle.yaml` and not the readme. The other test failure may also be a red hearing because I don't have the `tests/tests.yaml` file on this system. Here are the results from the run:

[bundletester_results.txt](https://github.com/juju-solutions/bundle-canonical-kubernetes/files/653455/bundletester_results.txt)

Since my manual tests worked so well I am willing to push this bundle so we can run some automated tests and see if we can reproduce these errors.

**Pull request merged**